Asgard.Console/ServiceMode.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Asgard.Communications;
+using Asgard.EngineControl;
+using Terminal.Gui;
+
+namespace Asgard.Console
+{
+    internal class ServiceMode : Window
+    {
+        private readonly EngineManager engineManager;
+        private readonly TextField cv;
+        private readonly TextField value;
+        private IEngineSession session;
```

Comment:
The `session` field should really be declared nullable.

Reply:
Fixed.